crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -313,7 +356,8 @@ async fn get_filtered_config(
                 .map_or_else(|_| json!(value), |int_val| json!(int_val)),
```

Comment:
I think we can now deprecate this handler.

Reply:
why?